### PR TITLE
Update openstack-ansible SHA to latest, closes #899

### LINKED ISSUE #899: Update SHAs for r11.1.6
##### What version (commit) do you encounter this issue on?
`r11.1.6`

##### What type of issue is this (bug, feature, documentation, regression)?
Minor release

##### What is the expected behavior?
N/A

##### What is actually happening?
N/A

##### How can we reproduce this issue?
N/A

##### How can we verify your issue is addressed?
N/A

##### If this is a large issue, what can be done immediately to alleviate the issue while working on a longer-term solution?
N/A

##### What is your evaluation on the importance of this issue, and why?
High

##### What branches / milestones do you believe this issue should target?
- [x] kilo

### PR DESCRIPTION
**Pull request by d34dh0r53**
This updates the openstack-ansible SHA to the latest
as of 18/03/2016.

Closes: #899

### CODE REVIEW

**Comment by prometheanfire** [comment]
ftfy, osa is now capping ansible for us

**Comment by prometheanfire** [comment]
recheck_swift
recheck_ceph

**Comment by d34dh0r53** [comment]
recheck_swift

**Comment by d34dh0r53** [comment]
recheck_ceph

**Comment by prometheanfire** [comment]
recheck_ceph - 15:01:39 jrpcaioiad-e1_galera_container-2edca90b : ok=25   changed=15   unreachable=0    failed=1   

recheck_swift - 14:54:54 jrpcaioiad-634_galera_container-54d2d9d9 : ok=25   changed=15   unreachable=0    failed=1

**Comment by prometheanfire** [comment]
recheck_swift - jrpcaioiad-9b4_galera_container-bf2f5a71 : ok=25   changed=15   unreachable=0    failed=1

recheck_ceph - jrpcaioiad-6eb_galera_container-e862c62a : ok=25   changed=15   unreachable=0    failed=1

**Comment by prometheanfire** [comment]
recheck_swift
recheck_ceph

**Comment by prometheanfire** [comment]
19:24:12 TASK: [galera_client | Install pip packages] ********************************** 
19:24:12 failed: [jrpcaioiad-3c0_galera_container-8dd3b77a] => (item=MySQL-python) => {"attempts": 5, "cmd": "/usr/local/bin/pip install MySQL-python", "failed": true, "item": "MySQL-python"}
19:24:12 msg: Task failed as maximum retries was encountered
19:24:12 failed: [jrpcaioiad-3c0_galera_container-8dd3b77a] => (item=python-memcached) => {"attempts": 5, "cmd": "/usr/local/bin/pip install python-memcached", "failed": true, "item": "python-memcached"}
19:24:12 msg: Task failed as maximum retries was encountered
19:24:12 failed: [jrpcaioiad-3c0_galera_container-8dd3b77a] => (item=pycrypto) => {"attempts": 5, "cmd": "/usr/local/bin/pip install pycrypto", "failed": true, "item": "pycrypto"}
19:24:12 msg: Task failed as maximum retries was encountered

**Comment by prometheanfire** [comment]
we need to run repo-build I think

**Comment by prometheanfire** [comment]
So, we have a problem here, the pip.conf references 11.2.12, however that doesn't exist and won't unless tagged.  We have a problem with the repo-build basing the directories off of git tags vs what is defined in `playbooks/inventory/group_vars/all.yml` as the `openstack_release`.  At least so says @cloudnull

**Comment by odyssey4me** [comment]
Until rpc-repo has a built repository with the 12.0.9 tag this will not work as this gate job does a clone from rpc-repo. Either there needs to be a request to OSA to tag so that the current jenkins job to build the repo may be activated, or the jenkins job needs to be modified to allow it to build from something other than a tag.

**Comment by prometheanfire** [comment]
@odyssey4me why do you say 12.0.9?  the pip.conf said 11.2.12?

**Comment by odyssey4me** [comment]
@prometheanfire My mistake. The principle of the comment still counts, even if I got the tag # wrong.

**Comment by prometheanfire** [comment]
Ok, just confirming.  also, from what I understand @cloudnull ran the rebuild job already.  I think the job needs to be altered.

**Comment by prometheanfire** [comment]
forcing a repo-build as well now

**Comment by prometheanfire** [comment]
looks like we needed the pywbem fix, updated the sha to that

**Comment by prometheanfire** [comment]
looking better so far

    17:09:59 TASK: [galera_client | Install pip packages] ********************************** 
    17:09:59 changed: [jrpcaioiad-191_galera_container-ae614ba5] => (item=MySQL-python)
    17:09:59 changed: [jrpcaioiad-191_galera_container-ae614ba5] => (item=python-memcached)
    17:09:59 changed: [jrpcaioiad-191_galera_container-ae614ba5] => (item=pycrypto)

File: scripts/deploy.sh [hunk 1]
```diff
@@ -122,8 +122,14 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
     popd
   fi
 
-  # setup the infrastructure
-  run_ansible setup-infrastructure.yml
+  # setup the infrastructure, run repo build to build untagged release
+  run_ansible memcached-install.yml
+  run_ansible repo-server.yml
+  run_ansible repo-build.yml
```
Comment (odyssey4me) [inline scripts/deploy.sh]: This is a diversion from how QE/support execute the builds in production. Please ensure that this does not merge.

Comment (prometheanfire) [inline scripts/deploy.sh]: I think it's probably something that's needed though

**Comment by prometheanfire** [comment]
the change to that file was the following

1. explicitly break out the `setup-infrastructure.yml` play into it's component parts
2. replace `repo-clone-mirror.yml` with repo-build.yml`

**Comment by prometheanfire** [comment]
abandoning til 11.2.12 is OSA tagged

**Pull request closed without merging**